### DISCUSSION TITLE
size-limit fails with rxjs imports

### DISCUSSION THREAD

**Discussion by Mihail04** [ai/size-limit#324] [General]
For angular library created with ng-packagr.
If the mjs file is import some from 'rxjs/operators', throw error
![image](https://github.com/ai/size-limit/assets/20125645/2d2d8c17-29dc-4a6a-b814-bd80142d9e87)
What could be the problem?

**Comment by ai** [comment]
Seems like, it’s webpack/esbuild issue with ESM import.

Hard to say from my side.
